<!-- HMI launch file -->

<launch>
	<group ns="RC_HMI">
		<node pkg="rc_hmi" name="HMI" type="rc_hmi">
			<param name="image_sub" value="/rcCamera/image_raw"/> 
			<param name="KukaCmdServiceName" value="/KukaNode"/> 
			<param name="PG70CmdServiceName" value="/PG70/PG70"/> 
		</node>
	</group>
</launch>
